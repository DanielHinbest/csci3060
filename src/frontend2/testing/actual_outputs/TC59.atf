00_levi____________FS_005000.00
