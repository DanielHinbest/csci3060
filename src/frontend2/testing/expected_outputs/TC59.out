Login successful.
Enter search string: No results found.
